### PR TITLE
Initialize with content from first draft

### LINKED ISSUE #2: Need initial content
Suggest someone open a PR to add content to the index.html

### PR DESCRIPTION
**Pull request by rhiaro**
Content copied from https://docs.google.com/document/d/1rYdWiwawWmLOWtHRvT0GzYcdewW_OS9M2mAkENLFdtY/edit#

I wasn't sure about authors/editors list, and there were a couple of internal refs in the original doc that didn't exist, so I left those deliberately broken for respec to pick them up as errors so we remember to fix them.

Fixes #2

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-rubric/pull/4.html" title="Last updated on Jul 8, 2020, 12:56 PM UTC (d3c1a22)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-rubric/4/6e316ad...d3c1a22.html" title="Last updated on Jul 8, 2020, 12:56 PM UTC (d3c1a22)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
I wonder why the reference generates the URL `https://w3.org/TR/did-core` for did core instead of `https://www.w3.org/TR/did-core`. It works, but that is not the official URL for the document. If this is generated by respec automatically, then there may be a problem somewhere (possibly in the specref list) and this should be tracked down.

**Comment by rhiaro** [comment]
@iherman that's my bad, I mistyped it in the local biblio, now fixed

File: index.html [hunk 1]
```diff
@@ -11,33 +11,3186 @@
...
+        wgURI: "https://www.w3.org/2019/did-wg/",
         wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/117488/status",
         github:                  {
             repoURL: "https://github.com/w3c/did-rubric/",
             branch: "master"
         },
-        noRecTrack: "true"
+        noRecTrack: "true",
+        editors: [{
+          name: "Joe Andrieu"
```
Comment (jandrieu) [inline index.html:23]: Update for Joe:
          name: "Joe Andrieu",
          company: "Invited Expert",
          url: "http://legreq.com",
          w3cid: 97261
        },{
          name: "Daniel Hardman",
          url: "https://github.com/dhh1128",
          company: "Evernym",
          companyURL: "http://evernym.com",

File: index.html [hunk 2]
```diff
@@ -11,33 +11,3186 @@
...
+        }],
+        authors: [{
+          name: "Joe Andrieu"
```
Comment (jandrieu) [inline index.html:26]: Ditto. Please use information from above.

File: index.html [hunk 3]
```diff
@@ -11,33 +11,3186 @@
...
+and many of the possible responses are not necessarily good or bad. It is up
+to the <a>Evaluator</a> to understand how each response in each criteria might
+illuminate favorable or unfavorable consequences for their needs.
+      </p>
+      <p>
+While this rubric allows the evaluation of many aspects of decentralization of
+a DID Method, it is not exhaustive, and does not cover other factors that may
+affect selection or adoption of a particular Method, such as privacy, security,
+reliability, or efficiency.
       </p>
```
Comment (jandrieu) [inline index.html:109]: <p>
The first draft of this document was based on _The Rubric for the Decentralization of DID Methods_, by Andrieu, et. al, published online at https://github.com/WebOfTrustInfo/rwot9-prague/blob/master/final-documents/decentralization-rubric.pdf
</p>

File: index.html (hunk 1 above)
Comment (dhh1128) [reply index.html:23]: I believe my w3cid = 108790. The url for me should probably be my LinkedIn profile: https://www.linkedin.com/in/danielhardman/. The other values are good.

File: index.html (hunk 2 above)
Comment (dhh1128) [reply index.html:26]: I'll propose that I be included as an author when I submit a meaningful content PR.

**Review by dhh1128 — APPROVED** [review]
LGTM as a beginning.

File: index.html (hunk 2 above)
Comment (jandrieu) [reply index.html:26]: That sounds right.

**Comment by dhh1128** [comment]
@jandrieu and @rhiaro : I raised a PR against this PR (see #5). If that PR is merged, then the changes Joe requested in the comment stream above will be included in this PR (as a separate commit, I believe) when it is merged.

**Pull request merged**